src/FerriteSolvers.jl

```diff
@@ -1,8 +1,11 @@
 module FerriteSolvers
+import LinearAlgebra
 export solve_ferrite_problem!
 
 export FerriteSolver
 export NewtonSolver
+export SteepestDescent
```

Comment:
```suggestion
export NewtonSolver, SteepestDescent
```
Minor change to show they are the same?